src/main/java/seedu/dukeofbooks/command/CheckBorrowingStatusCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.dukeofbooks.command;
+
+import seedu.dukeofbooks.controller.LoanController;
+import seedu.dukeofbooks.data.book.BorrowableItem;
+import seedu.dukeofbooks.data.exception.LoanRecordNotFoundException;
+
+public class CheckBorrowingStatusCommand extends UserCommand {
+    public static final String COMMAND_WORD = "status";
+    public static final String INCORRECT_SYNTAX = "INVALID SYNTAX\nExpected: status -title TITLE";
+    private static final String NOT_FOUND_MSG = "Item not found!";
```

Comment:
The strings used here seem to have some overlap with item availability. You might want to consider moving these into the messages.java file under commons